src/rp2_common/pico_bootrom/include/pico/bootrom.h

```diff
@@ -60,7 +61,72 @@ void *rom_data_lookup(uint32_t code);
  */
 bool rom_funcs_lookup(uint32_t *table, unsigned int count);
 
-typedef void __attribute__((noreturn)) (*reset_usb_boot_fn)(uint32_t, uint32_t);
+// The following is a list of bootrom functions available via rom_func_lookup and their signature
+
+typedef uint32_t (*rom_popcount32_fn)(uint32_t);
+#define ROM_FUNC_POPCOUNT32             rom_table_code('P','3')
```

Comment:
Could / should these `#define`s also be used in https://github.com/raspberrypi/pico-sdk/blob/master/src/rp2_common/pico_bit_ops/bit_ops_aeabi.S ?

Reply:
can't be because that is assembly